platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -689,9 +689,9 @@ OfflineDatabase::mergeDatabase(const std::string& sideDatabasePath) {
 
         // clang-format off
         mapbox::sqlite::Query queryRegions{ getStatement(
-            "SELECT r.id, r.definition, r.description "
-            "FROM side.regions sr "
-            "JOIN regions r ON sr.definition = r.definition") };
+                                                         "SELECT DISTINCT r.id, r.definition, r.description "
+                                                         "FROM side.regions sr "
+                                                         "JOIN regions r ON sr.definition = r.definition  AND sr.description IS r.description") };
```

Comment:
I defer to @mapbox/gl-core about this change.

Reply:
Looks good, but let's remove the added indentation to make it easier to read.